spec/features/user_sees_list_of_books_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+feature 'User sees a list of books' do
+  scenario 'successfully' do
+
+    books = create_list(:book, 3)
+
+    visit books_path
+
+    expect(page).to have_css('ul.books', :count => 1)
+    expect(page).to have_css('.books li', :count => 3)
```

Comment:
In this case I would test that the books titles are on the page. Something like

```ruby
books.each do |book|
  expect(page).to have_css('.books', text: book.title)
end
```

That way it's clear that you're testing that those books are on the page